.gitmodules

```diff
@@ -1,6 +1,10 @@
 [submodule "third_party/zlib"]
 	path = third_party/zlib
 	url = https://github.com/madler/zlib
+	# When using CMake to build, the zlib submodule ends up with a
```

Comment:
If I understand it correctly, this is mostly a cosmetic change so that `git status` doesn't show a distracting "modified content"  message for third_party/zlib.

Reply:
Yes, that's the purpose of this change. Shall I amend the commit message to be more explicit about this?